_posts/2017-01-28-mfem-3.3.md

```diff
@@ -0,0 +1,31 @@
+---
+title: MFEM 3.3 Released
+---
+
+Version 3.3 of MFEM, a lightweight, general, scalable C++ library for finite element methods, is now available at: [http://mfem.org](http://mfem.org)
+
+The goal of MFEM is to enable high-performance scalable finite element discretization research and application development on a wide variety of platforms, ranging from laptops to [exascale supercomputers](https://exascaleproject.org/2016/11/11/ecp_co-design_centers).
+
+It has many features, including:
+
+- 2D and 3D, arbitrary order H1, H(curl), H(div), L2, NURBS elements.
+- Parallel version scalable to hundreds of thousands of MPI cores.
+- Conforming/nonconforming adaptive mesh refinement (AMR), including anisotropic refinement, derefenement and parallel load balancing.
+- Galerkin, mixed, isogeometric, discontinuous Galerkin, hybridized, and DPG discretizations.
+- Support for triangular, quadrilateral, tetrahedral and hexahedral elements, including arbitrary order curvilinear meshes.
+- Scalable algebraic multigrid, time integrators, and eigensolvers.
+- Lightweight interactive OpenGL visualization with the MFEM-based [GLVis](http://glvis.org) tool.
+
+Some of the [new additions in version 3.3](https://raw.githubusercontent.com/mfem/mfem/master/CHANGELOG) are:
+
+- Comprehensive support for the linear and nonlinear solvers, preconditioners, time integrators and other features from the [PETSc](https://www.mcs.anl.gov/petsc) and [SUNDIALS](https://computation.llnl.gov/projects/sundials/sundials-software) suites.
```

Comment:
True, you can link to this commit instead (or not): https://github.com/mfem/mfem/commit/1569425857975d7c961b97d09c300da09aad9f82

It is by design for now, but this will probably change.